tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -52,6 +52,8 @@ const reportByUUID = (uuid) => $(`li[data-record-id="${uuid}"]`);
 
 const patientName = () => $('.subject .name');
 const reportType = () => $('div[test-id="form-title"]');
+const filterResetButton = () => $('.sidebar-reset');
+const openFilterButton = () => $('.open-filter');
```

Comment:
you can use this selector: [sidebarFilterOpenBtn](https://github.com/medic/cht-core/pull/7773/files#diff-4baa4edb0385cc54b8f1b69ec7e2af40aeb8f7371818d053e728ee2703e586bcR27) or the function [openSidebarFilter](https://github.com/medic/cht-core/pull/7773/files#diff-4baa4edb0385cc54b8f1b69ec7e2af40aeb8f7371818d053e728ee2703e586bcR215).

Just in case, the reset button doesn't clear the search bar anymore. To clear it we have [clearSearch](https://github.com/medic/cht-core/blob/b8c2044c4f3d45688abd10955e692f7ee855ddd6/tests/page-objects/search/search.wdio.page.js#L15)

Reply:
Thanks!